automation/Extension/firefox/data/create_http_requests_table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS http_requests_ext(
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  crawl_id INTEGER NOT NULL,
+  visit_id INTEGER NOT NULL,
+  url VARCHAR(500) NOT NULL,
```

Comment:
Any special reason to set the size limit (500)?
My understanding is that sqlite will ignore it: https://sqlite.org/faq.html#q9